examples/gitlab/gitlab.yml

```diff
@@ -0,0 +1,141 @@
+name: gitlab
+labels:
+    app: gitlab
+replicas: 1
+containers:
+      - name: gitlab
```

Comment:
can you adjust indentation to 2 spaces in this section, similar to the way you have done in section of `services` ?